test/Serilog.Sinks.MSSqlServer.Tests/DatabaseFixture.cs

```diff
@@ -42,7 +42,7 @@ public DatabaseFixture()
         
         public void Dispose()
         {
-            DeleteDatabase();
+            //DeleteDatabase();
```

Comment:
Ah, makes sense :+1: I'm okay with leaving it around, perhaps this line could be deleted instead of commented out?